prover/config_test.go

```diff
@@ -182,6 +191,7 @@ func (s *ProverTestSuite) SetupApp() *cli.App {
 		&cli.BoolFlag{Name: flags.ProveUnassignedBlocks.Name},
 		&cli.Uint64Flag{Name: flags.RPCTimeout.Name},
 		&cli.Uint64Flag{Name: flags.ProverCapacity.Name},
+		&cli.Uint64Flag{Name: flags.MinProofFee.Name},
```

Comment:
you changed this to uint64 but i think it can overflow, because the fee is in ETH and we also change TTKO decimals to 18 again, so we need to handle any fees and rewards via strings and convert to BigInt now?